test/e2e/flowaggregator_test.go

```diff
@@ -427,6 +430,15 @@ func testHelper(t *testing.T, data *TestData, podAIPs, podBIPs, podCIPs, podDIPs
 		}
 	})
 
+	// ToExternalFlows tests the case, where Pod is deployed on the source Node and sending traffic to an external IP,
+	// the flow information is exported as IPFIX flow records.
+	t.Run("ToExternalFlows", func(t *testing.T) {
+		clientName, clientIPs, cleanupFunc := createAndWaitForPod(t, data, data.createNginxPodOnNode, "test-client-", controlPlaneNodeName())
+		defer cleanupFunc()
+		// skipping test on v6 clusters, for no test clusters with reliable external connectivity over v6
+		checkRecordsForToExternalFlows(t, data, clientName, clientIPs.ipv4.String(), getExternalServerIP(t, externalServer, false))
```

Comment:
`skipIfNotIPv4Cluster(t)` is needed here. Thanks for reminding me. 
Simplified to `getExternalServerIPv4`. Originally I thought it does not matter that much for the util function to be more comprehensive, but it does make more sense to remove the v6 part since we do not support v6 tests. Thanks!